### PR TITLE
Pyangbind and model generation fix

### PR DESCRIPTION
**Pull request by JoseIgnacioTamayo**
No longer skip the oc-bfd model because of Python-reserved word
No longer skip warnings due to unsued YANG import.

### CODE REVIEW

**Comment by JoseIgnacioTamayo** [comment]
I just signed the CLA. The check passes now.

**Comment by JoseIgnacioTamayo** [comment]
@xavier-contreras , please take a look when you have some time.

File: oc_config_validate/oc_config_validate/update_models.sh [hunk 1]
```diff
@@ -32,15 +32,11 @@ echo -n "__model_versions__ = \"\"\"" > ${MODELS_FOLDER}/__init__.py
 for m in $( ls -d ${MODELS_PATH}/*/ ); do
   MODELS=$(find $m -name openconfig-*.yang)
   MODEL_NAME=$(basename $m | tr "-" "_")
-  # Skip openconfig-bdf model until https://github.com/robshakir/pyangbind/issues/286 is fixed
-  if [[ $MODEL_NAME == "bfd" ]]; then
-    continue
-  fi
   echo "Binding models in $m"
   # Skipping Warnings due to https://github.com/openconfig/public/issues/571
```
Comment (xavier-contreras) [inline oc_config_validate/oc_config_validate/update_models.sh:36]: I guess we can clean up this comment.

**Review by xavier-contreras — APPROVED** [review]
Minor nit, but LGTM

File: oc_config_validate/oc_config_validate/update_models.sh (hunk 1 above)
Comment (JoseIgnacioTamayo) [reply oc_config_validate/oc_config_validate/update_models.sh]: Done!

**Pull request merged**